tests/Client/UpdaterTest.php

```diff
@@ -314,6 +315,262 @@ public function providerVerifiedDelegatedDownload(): array
                     'level_3_below_terminated' => 1,
                 ],
             ],
+            // A terminating role only has an effect if the target path matches
+            // the role, otherwise the role is not evaluated.
+            // Roles after terminating delegation where the target path does match not
```

Comment:
"After" is an ambiguous term here. Would "below" maybe be more accurate?

Reply:
No "below" to be would mean roles the role with the terminating delegation delegated to it self.

If you look at this tree

```
#
#             Targets
#             /     \
#            a       f
#           / \
#          b   e
#         / \
#        c   d
#
```
If we are talking about "b" I would consider "c" and "d" "below". I would consider "e" after because it would be evaluated after b and it's delegations.

Reply:
Yeah, I agree with that interpretation. To make this clearer in the comment, what if we said "after (i.e., next to)"?

Reply:
Changed it.